src/main/scala/com/advancedtelematic/campaigner/db/Schema.scala

```diff
@@ -95,4 +98,30 @@ object Schema {
       ((CancelTask.apply _).tupled, CancelTask.unapply)
   }
   protected [db] val cancelTasks = TableQuery[CancelTaskTable]
+
+
+  type UpdatesTableRow = (UpdateId, String, UpdateType, Namespace, String, Option[String], Instant, Instant)
+
+  class UpdatesTable(tag: Tag) extends Table[Update](tag, "updates"){
+    def uuid = column[UpdateId]("uuid", O.PrimaryKey)
+    def updateId = column[String]("update_id")
+    def updateSourceType = column[UpdateType]("update_source_type")
+    def namespace = column[Namespace]("namespace")
+    def name = column[String]("name")
+    def description = column[Option[String]]("description")
+    def createdAt = column[Instant]("created_at")
+    def updatedAt = column[Instant]("updated_at")
```

Comment:
we generally don't need timestamps at the app level, I don't know why `Campaign` uses them. The db should have them so we can audit, but they are not needed on the app level.

Reply:
As we are going to publish our APIs (god help us), it is probably not a bad idea to have those for a campaign. It is not clear what "updated" means though...